maxim/logger/livekit/tts.py

```
@@ -49,58 +48,73 @@ def handle_tts_text_input(self, text):
-        session_info = None
-        if hasattr(self, "_session") and self._session is not None:
-            session_info = get_session_store().get_session_by_agent_session_id(
-                id(self._session)
-            )
+        if target_loop is None:
+            return None
+        return _run_channel_processing(result, target_loop, tts_id)
 
-        if session_info is None:
-            return
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)